static/js/background.js

```diff
@@ -2,32 +2,114 @@
 
 function getEvents()
 {
-    // localStorage['version_id'] = '';
-    console.log("called");
+    localStorage['version_id'] = '';
```

Comment:
My preference is to do getItem and setItem. 

I don't know what is better tho. :stuck_out_tongue: